charts/netdata/templates/cronjob.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if include "netdata.restarter.enabled" . }}
+---
+{{- $cmdList := list }}
+{{- $cmd := printf "kubectl rollout restart deployment %s-parent" (include "netdata.name" .) }}
+{{- $cmdList = append $cmdList (ternary $cmd "" .Values.parent.enabled) }}
+{{- $cmd = printf "kubectl rollout restart daemonset %s-child" (include "netdata.name" .) }}
+{{- $cmdList = append $cmdList (ternary $cmd "" .Values.child.enabled) }}
+{{- $cmd = printf "kubectl rollout restart deployment %s-k8s-state" (include "netdata.name" .) }}
+{{- $cmdList = append $cmdList (ternary $cmd "" .Values.k8sState.enabled) }}
+{{- $cmdList = compact $cmdList }}
```

Comment:
I couldn't find a way to make it less verbose. If you have any ideas, please suggest them.